frontend/public/actions/features.ts

```diff
@@ -164,7 +164,8 @@ export type FeatureAction = Action<
   typeof featureActions | typeof receivedResources | typeof clearFlags
 >;
 
-const openshiftPath = '/apis/apps.openshift.io/v1';
```

Comment:
The old API group only contains the `DeploymentConfig` resource which isn't a good candidate to test if its a OpenShift cluster since it's optional starting with 4.14.